packages/services/src/vckit.service.ts

```diff
@@ -44,9 +46,14 @@ export const issueVC = async ({
   credentialSubject,
   restOfVC,
   vcKitAPIUrl,
+  vcKitAPIKey,
 }: IVcKitIssueVC): Promise<VerifiableCredential> => {
   const body = constructCredentialObject({ context, type, issuer, credentialSubject, ...restOfVC });
-  const response = await publicAPI.post<VerifiableCredential>(`${vcKitAPIUrl}/credentials/issue`, body);
+  const response = await publicAPI.post<VerifiableCredential>(`${vcKitAPIUrl}/credentials/issue`, body, {
+    headers: {
+      ...vcKitAPIKey && { "Authorization": `Bearer ${vcKitAPIKey}` }
+    }
+  });
```

Comment:
Currently, the issuing VC is using the public API, eg: http://localhost:3332/v1/credentials/issue, so it doesn't need to add the API key

Reply:
Yep, we do if we want to use a VC service with an API key.

Reply:
Parking this PR for now.